### ISSUE TITLE
False positive DMLStatementInLoop

### ISSUE THREAD

**Issue by patrykacc** [Lightning-Flow-Scanner/lightning-flow-scanner-core#85]
https://github.com/Lightning-Flow-Scanner/lightning-flow-scanner-core/blob/e9f6a16eb47d2240701245b8d985dc5f5cf247ef/src/main/rules/DMLStatementInLoop.ts#L34

If in referenced line of code there will be a DML statement, then scanner will raise an false positive DMLStatementInLoop issue. So basicly `noMoreValuesConnector` are seen as inside loop, which is no correct as of my knowledge.a

**Comment by RubenHalman** [comment]
@patrykacc you are right and this is seemingly the reason people are experiencing a [bug](https://github.com/Lightning-Flow-Scanner/lightning-flow-scanner-core/issues/86) . Are you open to committing this change and becoming a contributor?

**Comment by patrykacc** [comment]
@RubenHalman Sure, I am glad I could help.
